bundler/helpers/Gemfile

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
```

Comment:
nest this under `helpers/v1` to make it explicit which version is `BUNDLED WITH`